### DISCUSSION TITLE
Interne Nummern ändern

### DISCUSSION THREAD

**Discussion by jwgn** [Freetz-NG/freetz-ng#1081] [Off-topic]
Hallo,

gibt es eine Möglichkeit, die internen Nummern angemeldeter Telefoniegeräte zu ändern?

Scheinbar werden die in `/var/flash/phonebook:nid` gespeichert. Ist es gefahrlos möglich, dort händisch Änderungen vorzunehmen?

**Comment by fda77** [comment]
Das sind alles closed-source Sachen von AVM,
Unter /var/flash/ kann man eigentlich nichts kaputt machen, durch ein recovery wird das zurückgesetzt.
vi oder nano funktionieren in dem Verzeichnis nicht, sonden von freetz `nvi`

**Reply by jwgn** [reply]
Alles klar, danke 👍
